lib/cloud-metadata/azure.js

```diff
@@ -1,32 +1,31 @@
 'use strict'
+const URL = require('url').URL
 const { httpRequest } = require('../http-request')
-const { cleanupRequest } = require('./shared')
-function getLogger () {
-  return require('../..').logger
-}
-
+const DEFAULT_BASE_URL = new URL('/', 'http://169.254.169.254:80')
```

Comment:
style nit: blank lines between require-block, var-block, and start of comment for the getMetadataAzure function would be good.

Reply:
happy to make this change -- but again -- let's think about formalizing these rules into our linter.

Reply:
✅